pkg/metrics/client_go_adapter.go

```diff
@@ -52,7 +52,21 @@ const (
 
 var (
 	// client metrics.
-	requestLatency = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+
+	// RequestLatency reports the request latency in seconds per verb/URL.
```

Comment:
Please document that this is deprecated, going to get removed and what the issue with it is.

Reply:
Done.